go/dist/dist_test.go

```diff
@@ -175,8 +178,7 @@ func testExec(t *testing.T, threshold, nnodes int) {
 }
 
 // Exec a child as a separate process.
-func testExecChild(t *testing.T, conf *testConfig, ctx context.Context,
-	grp *sync.WaitGroup) (io.Writer, io.Reader) {
+func testExecChild(ctx context.Context, conf *testConfig, t *testing.T, grp *sync.WaitGroup) (io.Writer, io.Reader) {
```

Comment:
I would respect the nice formatting of this file and add a proper return here

Reply:
I don't understand what you would like to change here (besides the line-width).